src/main/scala/ai/privado/languageEngine/ruby/processor/RubyProcessor.scala

```diff
@@ -145,7 +145,7 @@ object RubyProcessor {
             s"${TimeMetric.getNewTime()} - Overlay done in \t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
           )
 
-          new SQLParser(cpg, sourceRepoLocation, ruleCache).createAndApply()
+          new SchemaParser(cpg, sourceRepoLocation, ruleCache).createAndApply()
```

Comment:
why did we removed `SQLParser`? 

Without sqlParser, how will we parse SQL files?

Reply:
Name of new file is SchemaParser which is complete Ruby specific.
Adding SQLParser too.

Reply:
Added SQL Parser back: https://github.com/Privado-Inc/privado-core/pull/694/commits/b2c7217755a9d656a2d7ff62a73d56471270c784